### ISSUE TITLE
Record 

### ISSUE THREAD

**Issue by legnaro** [olavopeixoto/plugin.video.brplay#173]
É possível adicionar o canal Record, eles tem uma plataforma chamada Play Plus, posso tentar implementar os canais do Play Plus mas preciso saber como vcs capturam as entradas das APIs, seria manualmente pelo Wireshark?

**Comment by guiworldtv2** [comment]
> É possível adicionar o canal Record, eles tem uma plataforma chamada Play Plus, posso tentar implementar os canais do Play Plus mas preciso saber como vcs capturam as entradas das APIs, seria manualmente pelo Wireshark?

aparentemente este projeto foi descontinuado.... parece que não possui mais atualizações...

**Comment by bruno-briner** [comment]
> 
> aparentemente este projeto foi descontinuado.... parece que não possui mais atualizações...

Não ter atualizações não necessariamente significa que foi descontinuado já que o addon está funcionando ainda perfeitamente para mim.

**Comment by bruno-briner** [comment]
> É possível adicionar o canal Record, eles tem uma plataforma chamada Play Plus, posso tentar implementar os canais do Play Plus mas preciso saber como vcs capturam as entradas das APIs, seria manualmente pelo Wireshark?

Eu usava o Wireshark para descobrir as requisições feitas para as APIs quando eu ainda colaborava no, hoje finado, plugin [globo.com](https://bitbucket.org/bruno_briner/repository.brazilian.xbmc-addons/src/master/plugin.video.globo.com/)

**Comment by olavopeixoto** [comment]
> É possível adicionar o canal Record, eles tem uma plataforma chamada Play Plus, posso tentar implementar os canais do Play Plus mas preciso saber como vcs capturam as entradas das APIs, seria manualmente pelo Wireshark?

sim, manualmente com algum proxy. Pode ser o wireshark, debug do chrome, fiddler, mitmweb, etc

**Comment by olavopeixoto** [comment]
> > É possível adicionar o canal Record, eles tem uma plataforma chamada Play Plus, posso tentar implementar os canais do Play Plus mas preciso saber como vcs capturam as entradas das APIs, seria manualmente pelo Wireshark?
> 
> aparentemente este projeto foi descontinuado.... parece que não possui mais atualizações...

não foi descontinuado, mas estou com pouco tempo pra atualizar.
